tasks/libs/common/utils.py

```diff
@@ -425,9 +425,14 @@ def get_go_version():
 
 def get_root():
     """
-    Get the root of the Go project
-    """
-    return check_output(['git', 'rev-parse', '--show-toplevel']).decode('utf-8').strip()
+    Get the root of the Go project by walking up from this file looking for go.work.
+    """
+    current = os.path.dirname(os.path.abspath(__file__))
+    while current != os.path.dirname(current):  # Stop at filesystem root
+        if os.path.isfile(os.path.join(current, 'go.work')):
```

Comment:
I would prefer something that was not tied to go.  Perhaps  isdirectory(... '.github'), 

I know the comment says "root of the Go project", but the name of the method and the file it is in are more generic than Go.

Now if we are gitless, but still want this, perhaps MODULE.bazel, which is the build root.

Reply:
I was focusing on the "root of the Go project" comment, but no strong preference on my side. 

Should I just KISS and hard-code 4 dirs up from `os.path.abspath(__file__)`?